libs/nx-remotecache-s3/src/lib/util.ts

```diff
@@ -1,5 +1,10 @@
 import type { CustomRunnerOptions } from 'nx-remotecache-custom';
 import type { S3Options } from './setup-s3-task-runner';
+import { isMatch } from 'matcher';
+
+const HTTP_PROXY = 'HTTP_PROXY';
```

Comment:
My thought here was those env vars have that prefix because they are specific to this library.  `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` vars are not specific to this library however, as they are standard variables that are present in many environments.  Unless, I am mistaking what the prefix is for, I don't think we should look for non standard env vars that represent standard ones.

Reply:
Ah yes, I never used those variables myself but I was thinking in that direction as well. 
I suppose there would not be any use case for enabling the proxy for this library and not for others or vice versa?

Reply:
Right.  I can't imagine a scenario where you would want to use a proxy only for requests going through this library.  However, we could also look for NXCACHE_S3_* in addition to the standard variables.  It wouldn't be that hard to add, but may be a feature that would never be used.  We could wait to see if anyone requests it.  Your call.

Reply:
I agree, probably not necessary and it can still be added afterwards.